Assets/Plugins/iOS/Shopify/Checkout/Checkout.TitleView.swift

```diff
@@ -24,47 +24,47 @@
 //  THE SOFTWARE.
 //
 
+
 import UIKit
 
 extension Checkout {
 
-// Encapsulates the changing of the colors and state of the bottom button.
-class Button: UIButton {
-    var checkoutState: Checkout.State {
-        didSet {
-            updateButton()
-        }
-    }
-    
-    override var isHighlighted: Bool {
-        get {
-            return super.isHighlighted
-        }
-        set (value) {
-            super.isHighlighted = value
-            updateButton()
-        }
-    }
+class TitleView: UIView {
+    private let titleLabel: UILabel
+    private let lockImageView: UIImageView
     
-    init(cornerRadius: CGFloat) {
-        checkoutState = .checkingOut
-        super.init(frame: CGRect.zero)
-        setTitleColor(.white, for: .normal)
-        layer.cornerRadius = cornerRadius
-        translatesAutoresizingMaskIntoConstraints = false
-        titleLabel?.font = UIFont.boldSystemFont(ofSize: 16)
+    override init(frame: CGRect) {
+        titleLabel = UILabel()
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        titleLabel.text = "checkout.shopify.com"
+        titleLabel.textColor = .darkGray
+        titleLabel.font = UIFont.systemFont(ofSize: 16)
+        titleLabel.sizeToFit()
```

Comment:
Is this necessary? Wouldn't an auto-layout pass with constraints assigned to the label automatically use it's intrinsic content size anyway?